--- a/src/font-manager/Collections.vala
+++ b/src/font-manager/Collections.vala
@@ -540,7 +540,7 @@
             var row = ((CollectionListRow) source.widget);
             var drag_icon = new Gtk.Label(row.item_label.label);
             drag_icon.add_css_class("FontManagerListRowDrag");
-            var gtk_drag_icon = (Gtk.DragIcon) Gtk.DragIcon.get_for_drag(drag);
+            var gtk_drag_icon = (Gtk.DragIcon) new Gtk.DragIcon.get_for_drag(drag);
             gtk_drag_icon.set_child(drag_icon);
             return;
         }
--- a/src/font-manager/FontList.vala
+++ b/src/font-manager/FontList.vala
@@ -669,7 +669,7 @@
             widget_set_name(drag_count, "FontManagerListDragCount");
             drag_icon.add_overlay(drag_count);
             drag_count.set_label(selected_items.length.to_string());
-            var gtk_drag_icon = (Gtk.DragIcon) Gtk.DragIcon.get_for_drag(drag);
+            var gtk_drag_icon = (Gtk.DragIcon) new Gtk.DragIcon.get_for_drag(drag);
             gtk_drag_icon.set_child(drag_icon);
             return;
         }
